src/Components/ImagesTable/ImageDetails.js

```diff
@@ -17,7 +17,7 @@ import { useSelector } from 'react-redux';
 
 import ClonesTable from './ClonesTable';
 
-import { useGetSourceListQuery } from '../../store/provisioningApi';
+import { useGetSourceListQuery as UseGetSourceListQuery } from '../../store/provisioningApi';
```

Comment:
On a first quick run through the changes - this error is weird. It's a hook, maybe not a react one, but an RTK Query one. Still there are several similar hooks in the code and it seem the linter doesn't have any problem with those. 

Looks like `ReviewStepTextLists.js` uses the very same hook without any errors or warnings. :thinking:

Reply:
Ah! Thanks! I'll double check what's happening here

Reply:
Good catch, thanks for double checking. I completely misunderstood the issue yesterday evening. That commit is now patched up!